ts/src/base/Exchange.ts

```diff
@@ -3782,6 +3783,15 @@ export default class Exchange {
         ];
     }
 
+    getArrayOfObjectsKey (objects, key: IndexType) {
```

Comment:
we need this method in other similar PRs

Reply:
ok but the name is not clear at all, if I saw only the invocation I would not have an idea about its functionality

Reply:
also it's not "correct" to say arrayOfObjects when you blindly return array[i][key] meaning that the value can be anything

Reply:
it is not arrayOfObjects per se, it is "array of objects' keys" . might not sound ideal, but plz rename to a better name if you find

Reply:
its like "collect values of keys from objects" , a bit harder to be a short name